lib/rouge/lexers/mmt.rb

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class MMT < RegexLexer
+      title 'mmt'
+      desc 'MMT Surface Syntax (uniformal.github.io)'
+      tag 'mmt'
+      aliases 'mmtx'
+      mimetypes 'application/x-mmt'
+      filenames '*.mmt','*.mmtx'
+
+      state :root do
+        rule %r/\s+/im, Text::Whitespace
+        rule %r/\/T .*?\u275a/im, Comment::Multiline
```

Comment:
So `/T ` can begin a multiline comment?

Reply:
Yes, there are multiple ways to enter comments in the MMT language: `// ... `, `/T ...`. They do have different meanings, but that's not important for the syntax highlighter.

Reply:
Understood!